src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/EgressTests.cs

```diff
@@ -211,6 +211,39 @@ public async Task SharedConcurrencyLimitTest()
                 });
         }
 
+        /// <summary>
+        /// Tests that turning off HTTP egress results in an error for dumps and logs (gcdumps and traces are currently not tested)
+        /// </summary>
+        [Fact]
+        public async Task DisableHttpEgressTest()
+        {
+            await ScenarioRunner.SingleTarget(
+                _outputHelper,
+                _httpClientFactory,
+                DiagnosticPortConnectionMode.Connect,
+                TestAppScenarios.AsyncWait.Name,
+                appValidate: async (appRunner, appClient) =>
+                {
+                    ProcessInfo processInfo = await appClient.GetProcessAsync(appRunner.ProcessId);
+                    Assert.NotNull(processInfo);
+
+                    // Dump Error Check
+                    ValidationProblemDetailsException validationProblemDetailsExceptionDumps = await Assert.ThrowsAsync<ValidationProblemDetailsException>(
+                        () => appClient.CaptureDumpAsync(appRunner.ProcessId, DumpType.Mini));
+                    Assert.Equal(HttpStatusCode.BadRequest, validationProblemDetailsExceptionDumps.StatusCode);
+                    Assert.Equal(StatusCodes.Status400BadRequest, validationProblemDetailsExceptionDumps.Details.Status);
+
+                    // Logs Error Check
+                    ValidationProblemDetailsException validationProblemDetailsExceptionLogs = await Assert.ThrowsAsync<ValidationProblemDetailsException>(
+                            () => appClient.CaptureLogsAsync(appRunner.ProcessId, TestTimeouts.LogsDuration, Extensions.Logging.LogLevel.None, WebApi.LogFormat.NDJson));
```

Comment:
nit: Add `using`s at the top of this file for `Microsoft.Extensions.Logging` and `Microsoft.Diagnostics.Monitoring.WebApi` and then remove the namespace disambiguation here. It makes it a little harder to read given the length of the code line.